tuna/parse_args.py

```diff
@@ -26,132 +26,133 @@
-                        dest='restart_machine',
-                        action='store_true',
-                        default=False,
-                        help='Restart machines')
-  if TunaArgs.DOCKER_NAME in arg_list:
-    parser.add_argument('--docker_name',
-                        dest='docker_name',
-                        type=str,
-                        default='miopentuna',
-                        help='Select a docker to run on. (default miopentuna)')
+          choices=[None, '1', '2', '4'])
+    if TunaArgs.CONFIG_TYPE in arg_list:
+      parser.add_argument('-C',
+                          '--config_type',
+                          dest='config_type',
+                          help='Specify configuration type',
+                          default=ConfigType.convolution,
+                          choices=[cft.value for cft in ConfigType],
+                          type=ConfigType)
+    # pylint: disable=duplicate-code
+    if TunaArgs.SESSION_ID in arg_list:
+      parser.add_argument('--session_id',
+                          dest='session_id',
+                          type=int,
+                          help='Session ID to be used as tuning tracker.\
+        Allows to correlate DB results to tuning sessions')
+    # pylint: enable=duplicate-code
+    if TunaArgs.MACHINES in arg_list:
+      parser.add_argument('-m',
+                          '--machines',
+                          dest='machines',
+                          type=str,
+                          default=None,
+                          required=False,
+                          help='Specify machine ids to use, comma separated')
+    if TunaArgs.REMOTE_MACHINE in arg_list:
+      parser.add_argument('--remote_machine',
+                          dest='remote_machine',
+                          action='store_true',
+                          default=False,
+                          help='Run the process on a network machine')
+    if TunaArgs.LABEL in arg_list:
+      parser.add_argument('-l',
+                          '--label',
+                          dest='label',
+                          type=str,
+                          default=None,
+                          help='Specify label for jobs')
+    if TunaArgs.RESTART_MACHINE in arg_list:
+      parser.add_argument('-r',
+                          '--restart',
+                          dest='restart_machine',
+                          action='store_true',
+                          default=False,
+                          help='Restart machines')
+    if TunaArgs.DOCKER_NAME in arg_list:
+      parser.add_argument(
+          '--docker_name',
+          dest='docker_name',
+          type=str,
+          default='miopentuna',
+          help='Select a docker to run on. (default miopentuna)')
 
   return parser
 
 
-def clean_args():
+def clean_args() -> None:
   """clean arguments"""
-  libs = [elem.value for elem in Library]
+  libs: List[Any] = [elem.value for elem in Library]
```

Comment:
i dont think it should be any, it should be List[Library] or something like that